### ISSUE TITLE
Improve chat question carousel accessibility

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#292917]
The chat question carousel will soon be more accessible thanks to https://github.com/microsoft/vscode/pull/292911.

However, it still lacks an aria label and one can not move focus via keyboard to the container.

Additionally, focus automatically moves to the first response, which is against our usual pattern and could be very disruptive and confusing for screen reader users.

I'm thinking when in screen reader mode, we should not move focus. Since they're alerted, we should add a command and keybinding to focus the first response.

cc @accesswatch, @jooyoungseo, @rperez030

**Comment by meganrogge** [comment]
cc @justschen

**Comment by meganrogge** [comment]
I have not assigned a keybinding, we need to discuss that.

**Comment by meganrogge** [comment]
I confirmed that:
- when in screen reader mode, we should NOT auto focus
- we should add a tab index to the container and an appropriate aria label (the current question)
- we should add a command and default keybinding to toggle focus between this and the chat input, `ctrlCmd+shift+a` to align with the confirmation dialog.
